File: server/database/quizgame.go
// Go connection Sample Code:
package main
import (
	_ "github.com/microsoft/go-mssqldb"
	"database/sql"
	"context"
	"log"
	"fmt"
	"errors"
)

var db *sql.DB
var server = "desarrollosoftware1.database.windows.net"
var port = 1433
var user = "admin123"
var password = "<your_password>"
var database = "quizGame"

func main() {
	// Build connection string
	connString := fmt.Sprintf("server=%s;user id=%s;password=%s;port=%d;database=%s;",
		server, user, password, port, database)
	var err error
	// Create connection pool
	db, err = sql.Open("sqlserver", connString)
	if err != nil {
		log.Fatal("Error creating connection pool: ", err.Error())
	}
	ctx := context.Background()
	err = db.PingContext(ctx)
	if err != nil {
		log.Fatal(err.Error())
	}
	fmt.Printf("Connected!")
}

File: server/database/quizgame.js
const sql = require('mssql');
const config = {
  user: 'admin123',
  password: 'Fernandot123',
  server: 'desarrollosoftware1.database.windows.net',
  database: 'quizGame',
  options: {
    trustedConnection: true
  }
};

function loadDatabase() {
  sql.connect(config).then(pool => {
    console.log('Connected to MSSQL');
    return pool;
  }).catch(err => {
    console.log('Database connection failed. Error in configuration: ', err);
  });
}

module.exports = { sql, loadDatabase };